src/ipc/capnp/chain.capnp

```diff
@@ -0,0 +1,195 @@
+
+using Common = import "common.capnp";
+using Handler = import "handler.capnp";
+
+interface Chain $Proxy.wrap("interfaces::Chain") {
+    destroy @0 (context :Proxy.Context) -> ();
+    getHeight @1 (context :Proxy.Context) -> (result :Int32, hasResult :Bool);
+    getBlockHash @2 (context :Proxy.Context, height :Int32) -> (result :Data);
+    haveBlockOnDisk @3 (context :Proxy.Context, height :Int32) -> (result :Bool);
+    getTipLocator @4 (context :Proxy.Context) -> (result :Data);
+    getActiveChainLocator @5 (context :Proxy.Context, blockHash :Data) -> (result :Data);
+    findLocatorFork @6 (context :Proxy.Context, locator :Data) -> (result :Int32, hasResult :Bool);
+    hasBlockFilterIndex @7 (context :Proxy.Context, filterType :UInt8) -> (result :Bool);
+    blockFilterMatchesAny @8 (context :Proxy.Context, filterType :UInt8, blockHash :Data, filterSet :List(Data)) -> (result :Bool, hasResult :Bool);
+    findBlock @9 (context :Proxy.Context, hash :Data, block :FoundBlockParam) -> (block :FoundBlockResult, result :Bool);
+    findFirstBlockWithTimeAndHeight @10 (context :Proxy.Context, minTime :Int64, minHeight :Int32, block :FoundBlockParam) -> (block :FoundBlockResult, result :Bool);
+    findAncestorByHeight @11 (context :Proxy.Context, blockHash :Data, ancestorHeight :Int32, ancestor :FoundBlockParam) -> (ancestor :FoundBlockResult, result :Bool);
+    findAncestorByHash @12 (context :Proxy.Context, blockHash :Data, ancestorHash :Data, ancestor :FoundBlockParam) -> (ancestor :FoundBlockResult, result :Bool);
+    findCommonAncestor @13 (context :Proxy.Context, blockHash1 :Data, blockHash2 :Data, ancestor :FoundBlockParam, block1 :FoundBlockParam, block2 :FoundBlockParam) -> (ancestor :FoundBlockResult, block1 :FoundBlockResult, block2 :FoundBlockResult, result :Bool);
+    findCoins @14 (context :Proxy.Context, coins :List(Common.Pair(Data, Data))) -> (coins :List(Common.Pair(Data, Data)));
+    guessVerificationProgress @15 (context :Proxy.Context, blockHash :Data) -> (result :Float64);
+    hasBlocks @16 (context :Proxy.Context, blockHash :Data, minHeight :Int32, maxHeight: Int32, hasMaxHeight :Bool) -> (result :Bool);
+    isRBFOptIn @17 (context :Proxy.Context, tx :Data) -> (result :Int32);
+    isInMempool @18 (context :Proxy.Context, txid :Data) -> (result :Bool);
+    hasDescendantsInMempool @19 (context :Proxy.Context, txid :Data) -> (result :Bool);
+    broadcastTransaction @20 (context :Proxy.Context, tx: Data, maxTxFee :Int64, relay :Bool) -> (error: Text, result :Bool);
+    getTransactionAncestry @21 (context :Proxy.Context, txid :Data) -> (ancestors :UInt64, descendants :UInt64, ancestorsize :UInt64, ancestorfees :Int64);
+    calculateIndividualBumpFees @22 (context :Proxy.Context, outpoints :List(Data), targetFeerate :Data) -> (result: List(Common.PairInt64(Data)));
+    calculateCombinedBumpFee @23 (context :Proxy.Context, outpoints :List(Data), targetFeerate :Data) -> (result :Int64, hasResult :Bool);
```

Comment:
Why are the `CFeeRate` serialized as `Data` whereas the other `CAmount` are `Int64`?